.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         run: uv python install 3.11
 
       - name: Install the dependencies
-        run: uv sync 
+        run: uv sync
 
       - name: Run pre-commit checks
         run: |
```

Comment:
let's fix your CI, you need to add this to your last step:

uv pip install pre-commit
uv run pre-commit run --all-files